pkg/routes/devicegroups.go

```diff
@@ -323,6 +323,10 @@ func GetAllDeviceGroups(w http.ResponseWriter, r *http.Request) {
 func CreateDeviceGroup(w http.ResponseWriter, r *http.Request) {
 	ctxServices := dependencies.ServicesFromContext(r.Context())
 
+	if feature.EdgeParityInventoryGroupsEnabled.IsEnabled() && !feature.EnforceEdgeGroups.IsEnabled() {
```

Comment:
feature.EnforceEdgeGroups.IsEnabled() is always enabled but using utility.EnforceEdgeGroups(orgID) will depend from orgid,  if orgid is in the list it's enabled if not it will disabled
```suggestion
	if feature.EdgeParityInventoryGroupsEnabled.IsEnabled() && !utility.EnforceEdgeGroups(orgID) {
```

Please update all similar conditions with this suggestion